samples/simple-chat/csharp/FunctionApp/Startup.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using FunctionApp;
+using Microsoft.Azure.Functions.Extensions.DependencyInjection;
+using Microsoft.Azure.WebJobs.Extensions.SignalRService;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.IdentityModel.Tokens;
+
+[assembly: FunctionsStartup(typeof(Startup))]
+namespace FunctionApp
+{
+    /// <summary>
+    /// Runs when the Azure Functions host starts. Microsoft.NET.Sdk.Functions package version 1.0.28 or later
+    /// </summary>
+    public class Startup : FunctionsStartup
+    {
+        public override void Configure(IFunctionsHostBuilder builder)
+        {
+            // Get the configuration files for the OAuth token issuer
+            //var issuerToken = Environment.GetEnvironmentVariable("IssuerToken");
```

Comment:
>//var issuerToken = Environment.GetEnvironmentVariable("IssuerToken"); [](start = 12, length = 70)

Remove unused code.

Reply:
Only a POC for now, I want to make it clean...

Reply:
I keep this line to show how can a customer use it. The next line is for easily run the sample without environment variable